index.html

```diff
@@ -325,5 +325,135 @@ <h2>About me</h2>
       </div>
     </section>
     <!-- end of about section -->
+
+    <!-- start of contact and footer sections -->
+    <section class="contact-section">
+      <h3 class="contact-section-header">
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in
+        touch.
+      </h3>
+
+      <form action="https://formspree.io/f/xeqvdwpz" method="POST">
+        <input
+          type="text"
+          name="name"
+          id="name"
+          placeholder="Full Name"
+          minlength="1"
+          maxlength="10"
```

Comment:
Maxlength should be 30. length of 10 characters is too short for names. The `minlength` is redundant here since we already have a required attribute